tests/int/collections/tags.int.spec.ts

```
@@ -0,0 +1,555 @@
+            contains: searchTerm
+          }
+        }
+      })
+
+      expect(result.docs.length).toBeGreaterThanOrEqual(2)
+    })
+
+    it('devrait trouver des tags par couleur', async () => {
+      const unique = createUniqueTestData()
+      // Generate a unique color based on the unique data to avoid conflicts
+      const colorSeed = parseInt(unique.slug.slice(-6), 36).toString(16).padStart(6, '0').slice(0, 6)
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)